cmd/data.go

```diff
@@ -125,7 +125,12 @@ func (cmd *DataCmd) Execute(ctx context.Context, f *flag.FlagSet, _ ...interface
 
 	// If filePrefix not explicitly set, use dbName as prefix.
 	if cmd.filePrefix == "" {
-		cmd.filePrefix = dbName + "."
+		dirPath := "harbour_bridge_output/" + dbName + "/"
+		err = os.MkdirAll(dirPath, os.ModePerm)
```

Comment:
Should we check if the directory with the same name already exists before making it?

Reply:
AFAIK, this function is idempotent. But best to verify.

Reply:
I verified this, it is idempotent.

Reply:
> Can you also add how does the user specify the `--prefix` flag now? 

Have added this in the PR description. 

> Seems like the `harbour_bridge_output` is only created when `--prefix` is not specified, and when it is, the generated files would still go the root directory?

The `prefix` flag is used to set file paths explicitly. If it is used then we don't have to put the files in the `harbour_bridge_output` directory.

>> Putting these files in a seperate directory will also be useful in mounting this directory to the host machine for HarbourBridge in Docker mode.
>
> For this objective, shouldn't the news files be created inside a directory irrespective of the `--prefix` flag?

Yes they should be. But we don't have a way to differentiate if the migration is happening in the normal mode or the Docker mode. We can document in the usage guide that the prefix flag is not relevant in the Docker Mode.